rasa/core/policies/ted_policy.py

```diff
@@ -688,16 +693,20 @@ def train(
                 f"file to avoid this warning.",
                 category=UserWarning,
             )
-            return
+            return self._resource
```

Comment:
💡 We return the `resource` after the training so that potential children nodes can receive the resource and then load the policy from the `model_storage` (we don't make use of this for policies but [we e.g. do that for featurization in the "nlu part" of the graph](https://www.notion.so/rasa/Rasa-3-0-Architecture-Scoping-Document-7dfb4454b436459f9eee0ec70d07aec7#5fdfa64cfa79436c8c5a1d06150b7565)).